trino.yaml

```
@@ -195,7 +192,7 @@ test:
         - ${{package.name}}-oci-entrypoint # need to setup basic
         - bash
     environment:
-      JAVA_HOME: /usr/lib/jvm/java-23-openjdk
+      JAVA_HOME: /usr/lib/jvm/java-24-openjdk
```

```suggestion
      JAVA_HOME: /usr/lib/jvm/default-jvm
```
this way we won't have to update it on jdk updates.

---

updated